tests/bootstrap.php

```diff
@@ -9,6 +9,6 @@
  * file that was distributed with this source code.
  */
 
-$loader = require_once __DIR__ . "/../vendor/autoload.php";
+$loader = require __DIR__ . "/../vendor/autoload.php";
```

Comment:
What is the reason for this change? I'd expect the autoload file to only be loaded once.